modules/exploits/windows/scada/rockwell_factorytalk_rce.rb

```diff
@@ -0,0 +1,239 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Powershell
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Rockwell FactoryTalk View SE SCADA Unauthenticated Remote Code Execution',
+        'Description' => %q{
+          This module exploits a series of vulnerabilities to achieve unauthenticated remote code execution
+          on the Rockwell FactoryTalk View SE SCADA product.
+          The default configuration is exploitable by an unauthenticated attacker, and the exploit runs as
+          the IIS user on the Windows sever.
+          The attack relies on the chaining of five separate vulnerabilities. The first vulnerability is an unauthenticated project copy request,
+          the second is a directory traversal, and the third is a race condition. In order to achieve full remote code execution on all
+          targets, two information leak vulnerabilities are also abused.
+          This exploit was used by the Flashback team (Pedro Ribeiro + Radek Domanski) in Pwn2Own Miami 2020 to win the EWS category.
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+        [
+          'Pedro Ribeiro <pedrib[at]gmail.com>',         # Vulnerability discovery and Metasploit module
```

Comment:
The spacing here is the only rubocop offense, would you mind changing it to the following please?

```suggestion
          'Pedro Ribeiro <pedrib[at]gmail.com>', # Vulnerability discovery and Metasploit module
```

Reply:
done